R/package2readme.R

```diff
@@ -161,13 +161,12 @@ package2readme <- function(package = ".", add = NULL) { # nolint
     for (a in add) {
       if (file.exists(file.path(folder, a))) a <- file.path(folder, a)
       if (file.exists(a)) {
-        fType <- tail(strsplit(a, "\\.")[[1]], 1)
+        fileType <- tools::file_ext(a)
         if (tolower(fType) == "rmd") {
```

Comment:
`fType` needs to be renamed everywhere, not sure if this is the only place.
```suggestion
        if (tolower(fileType) == "rmd") {
```